lib/server/push.api.js

```diff
@@ -320,11 +332,14 @@ function serverSend(raixNotification) {
 }
 
 function querySend(query, raixNotification) {
-  console.log('Push: Sending notifications for query: ', query);
+  console.log('Push: Sending notifications for query: ', JSON.stringify(query));
 
   let firebasePromises = [];
 
-  let appTokens = AppTokenUtil.getAppTokensBasedOnQuery(query);
+  let desktopAppId = raixNotification && raixNotification.payload && raixNotification.payload.isDesktop && raixNotification.payload.vetId || undefined;
```

Comment:
@mjovkovic It makes more sense to keep `isDesktop` flag directly in `raixNotification` instead in `raixNotification.payload`.